chopsticks-scripts/run_migration.ts

```diff
@@ -33,6 +35,20 @@ const { polkadot, assetHub } = await setupNetworks({
     },
 });
 
+await fetchStorages({
+    block: process.env.POLKADOT_ASSET_HUB_BLOCK_NUMBER_PRE,
+    endpoint: process.env.POLKADOT_ASSET_HUB_ENDPOINT || `ws://localhost:${process.env.AH_NODE_RPC_PORT}`,
+    dbPath: "./dbs/polkadot-asset-hub.sqlite",
+    config: ['0x'],
+});
+
+await fetchStorages({
+    block: process.env.POLKADOT_BLOCK_NUMBER_PRE,
+    endpoint: process.env.POLKADOT_ENDPOINT || `ws://localhost:${process.env.RELAY_NODE_RPC_PORT}`,
```

Comment:
This is not critical, but if both of these env vars are unset, we'll get `ws://localhost:undefined`, which may or may not be immediately identifiable as the cause of the script's failure.

This a little more verbose, but would avoid ⬆️:

```ts
endpoint: process.env.POLKADOT_ENDPOINT || (() => {
  if (!process.env.RELAY_NODE_RPC_PORT) {
    console.error('RELAY_NODE_RPC_PORT required when POLKADOT_ENDPOINT not set');
    process.exit(1);
  }
  return `ws://localhost:${process.env.RELAY_NODE_RPC_PORT}`;
})()
```